src/components/layout.js

```diff
@@ -20,6 +20,8 @@ const LayoutContainer = styled.div`
     margin: 0 auto;
     max-width: 750px;
 
+    max-width: ${props => props.wide && '1200px'};
```

Comment:
there's a `max-width` on line 21... delete that one?

Reply:
switched to a ternary between 1200px and 750px. Better?